component-samples/demo/aio/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+FROM ubuntu:20.04
+ARG _JAVA_OPTIONS
+ENV _JAVA_OPTIONS=${_JAVA_OPTIONS}
+RUN apt-get update && apt-get install -y openjdk-11-jdk libsofthsm2 softhsm softhsm-common opensc wget haveged
```

Comment:
Should include `apt-get -y upgrade` to ensure we have latest packages installed in the docker container.